siliconcompiler/apps/sc_dashboard.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Silicon Compiler Authors. All Rights Reserved.
+import sys
+import siliconcompiler
+
+
+def main():
+    progname = "sc-dashboard"
+    description = """
+    --------------------------------------------------------------
+    Restricted SC app that opens a dashboard...
```

Comment:
Need to fix this:
Should probably be expanded to say "... a dashboard with the provided manifest" along with Examples for how to launch.
```
>>> sc-dashboard -cfg <build>
```